web_src/js/components/RepoActionView.vue

```diff
@@ -286,51 +286,48 @@ export function initRepositoryActionView() {
 
 </script>
 
-<style scoped lang="less">
-
+<style scoped>
 .action-view-body {
   display: flex;
-  height: calc(100vh - 266px); // fine tune this value to make the main view has full height
+  height: calc(-166vh);
```

Comment:
`height: calc(-166vh);` ?

Reply:
Interesting bug. I ran the Less in Vue through https://jsonformatter.org/less-to-css and it does reproduce the bug.

Reply:
Hopefully there isn't other `calc` bug anywhere else.

Reply:
Will re-run through `lessc` which does not show that bug

```bash
$ echo 'a{height: calc(100vh - 266px)}' > x.less
$ npx lessc x.less
a {
  height: calc(100vh - 266px);
}
```

Reply:
All `.less` files were ran though `lessc`, so only the styles in Vue components can be affected.

Reply:
Rebuilt it, it's fixed and comments are restored as well.